.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+	// See http://go.microsoft.com/fwlink/?LinkId=827846
+	// for the documentation about the extensions.json format
+	"recommendations": [
+		"dbaeumer.vscode-eslint"
+	]
+}
```

Comment:
This file is not valid json. Also, it lists a set of extensions to be recommended with ours. I do not think we can recommend anything so please delete the file.